tmva/tmva/src/TSpline2.cxx

```diff
@@ -51,10 +51,7 @@ TMVA::TSpline2::TSpline2( const TString& title, TGraph* theGraph )
 ////////////////////////////////////////////////////////////////////////////////
 /// destructor
 
-TMVA::TSpline2::~TSpline2( void )
-{
-   if (fGraph) delete fGraph; // ROOT's spline classes also own the TGraph
-}
+TMVA::TSpline2::~TSpline2(void) {}
```

Comment:
Suggestion: extend on line 42 the documentation, saying that TGraph is not owned by TSpline

Reply:
Updated.